test/utils/assert.js

```diff
@@ -1,18 +1,6 @@
 'use strict';
 
-const assert = require('assert');
-
-function deepEqual(result, expected, message) {
-
-    try {
-        assert.deepEqual(result, expected, message);
-    } catch (e) {
-        console.log(`Expected:\n${JSON.stringify(expected, null, 2)}`);
-        console.log(`Result:\n${JSON.stringify(result, null, 2)}`);
-        throw e;
-    }
-
-}
+const { assert } = require('chai');
```

Comment:
done, except for assert.fails() since its just a function and not a real assertion i wasnt sure how best to deal with it. do you think I should keep as-is, or do something to chai-ify it?